docs/design/autoware-interfaces/ad-api/types/autoware_ad_api_msgs/srv/lanelet_route_set.md

```diff
@@ -0,0 +1,17 @@
+# autoware_ad_api_msgs/srv/LaneletRouteSet
+
+## Definition
+
+```txt
+std_msgs/Header header
+geometry_msgs/Pose[<=1] start
+geometry_msgs/Pose goal
+autoware_ad_api_msgs/LaneletSegment[] segments
```

Comment:
@mitsudome-r Does the word `LaneletSegment` make sense for this usage?
(Although I'm not a native speaker) I feel `RouteSegment` is acceptable, but `LaneletSegment` is a bit weird. :thinking:

Reply:
@mitsudome-r Could you confirm this? :pray:

Reply:
I believe Route Segment is better. Lanelet segment is used for different purpose in Lanelet library.